packages/plugin-ext/src/main/browser/editors-and-documents-main.ts

```diff
@@ -168,6 +169,19 @@ export class EditorsAndDocumentsMain implements Disposable {
     saveAll(includeUntitled?: boolean): Promise<boolean> {
         return this.modelService.saveAll(includeUntitled);
     }
+
+    hideEditor(id: string): Promise<void> {
+        for (const editorWidget of this.editorService.all) {
```

Comment:
Before trying to remove this loop, is there an existing way to get the EditorWidget for a MonacoEditor that I have missed?

Reply:
Since the editor data reported to the plugin side has gone through the `EditorSnapshot` system, I don't really see a way recover the editor meant other than to repeat the process.